chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -953,17 +953,14 @@ def dataset_match(datasets, device_path):
                     if action & self.CZP_REMOVEZPOOLS:
                         self.execute_simultaneous_commands(zfs_device.clear_device_commands,
                                                            [server['fqdn'] for server in test_servers],
-                                                           'recursive destroy zpool %s' % zfs_device)
-
-                        # ldiskfs_device = TestBlockDevice('linux', first_test_server['device_paths'][lustre_device['path_index']])
-                        #
-                        # self.execute_simultaneous_commands(ldiskfs_device.destroy_commands,
-                        #                                    [first_test_server['fqdn']],
-                        #                                    'clearing disk because zfs import failed %s' % ldiskfs_device,
-                        #                                    expected_return_code=0)
-                        #
-                        # [self.remote_operations.reset_server(server['fqdn']) for server in test_servers]
-                        # [self.remote_operations.await_server_boot(server['fqdn']) for server in test_servers]
+                                                           'recursive destroy zpool %s' % zfs_device,
+                                                           expected_return_code=None)
+
+                        ldiskfs_device = TestBlockDevice('linux', first_test_server['device_paths'][lustre_device['path_index']])
+
+                        self.execute_simultaneous_commands(ldiskfs_device.wipe_signatures_commands,
```

Comment:
Why are we doing a `ldiskfs` device type "wipe" on what is supposed to be a ZFS disk?

Is `cleanup_zfs_pools()` being called speculatively on devices which we don't know the contents of or is it only being called for disks which we know have ZFS on them?  I.e. because we put it there.